apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/ClientPool.cs

```diff
@@ -229,7 +229,29 @@ public SpannerClientCreator(ClientCredentialKey parentKey)
                     channelCredentials);
                 Logger.LogPerformanceCounterFn("SpannerClient.RawCreateCount", x => x + 1);
 
-                return SpannerClient.Create(channel);
+                //Pull the timeout from spanner options.
+                //The option must be set before OpenAsync is called.
+                var callSettings = CallSettings.FromCallTiming(
+                    CallTiming.FromRetry(
+                        new RetrySettings(
+                            SpannerSettings.GetDefaultRetryBackoff(),
+                            SpannerSettings.GetDefaultTimeoutBackoff(),
+                            Expiration.FromTimeout(SpannerOptions.Instance.Timeout),
+                            SpannerSettings.NonIdempotentRetryFilter
+                        )));
+
+                return SpannerClient.Create(
```

Comment:
these got moved around/refactored for the pool tests (to make a replaceable clientfactory for testability).  so I'll fix it there...